### ISSUE TITLE
Archived entrys wont return

### ISSUE THREAD

**Issue by KoEnix** [evait-security/envizon#76]
Hello again,

while working with envizon, it happend to me that I accidentlly archived an entry. But I wasnt able to restore it back to a normal entry, after being archived.
The version, as far as I can say, is 4.0 (its written right under "Settings").
I hope, you can reproduce and fix this issue. 

Best regards,
Chris

**Comment by blacklist-arcc** [comment]
An archived client should be moved in a "fake" group. It's the last entry in the groups view (left sidebar) and should be named archived clients. You can unarchive clients by clicking in this fake group and using the top buttons

**Comment by blacklist-arcc** [comment]
If not, you may have to update the container. (Data will be not lost)

```
cd [FOLDER CONTAINS COMPOSE FILE]
docker-compose pull envizon
docker-compose up --force-recreate -d envizon
```

**Comment by blacklist-arcc** [comment]
fixed in 28682f95addff4163b5818fe1e18cdb45c57d649
